backend/engine/plugins/trivy_sca/main.py

```diff
@@ -38,7 +38,7 @@ def main():
     results = []
 
     # Generate Lock files
-    lock_file_errors, lock_file_alerts = check_package_files(args.path, include_dev)
+    lock_file_errors, lock_file_alerts = check_package_files(args.path, include_dev, False)
```

Comment:
NW, we don't need npm packages to be installed for trivy_sca because license info is irrelevant, in SBOM we need to install to get license info. I believe my previous comment about this naming can help clear up this confusion

Reply:
yup, makes sense, maybe just leave code comments explaining as much